tests/test_wind_turbine.py

```diff
@@ -48,13 +48,15 @@ def test_get_turbine_types(self, capsys):
         with pytest.raises(ValueError, match=msg):
             get_turbine_types("wrong")
 
+    def test_store_turbine_data_from_oedb(self):
+        store_turbine_data_from_oedb()
+
```

Comment:
This doesn't test anything, does it?

Reply:
I did not change this test, I just moved it. It is a pure function test. 

Now I added a test that checks if the files have been updated by the function call.